src/library/Uno.Cupertino/Styles/Controls/TextBlock.xaml

```diff
@@ -1,148 +1,150 @@
 ﻿<ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
 					xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml">
-	
+
 	<ResourceDictionary.MergedDictionaries>
 		<ResourceDictionary Source="ms-appx:///Styles/Application/Fonts.xaml" />
 	</ResourceDictionary.MergedDictionaries>
 
-	<!-- Based on https://developer.apple.com/design/human-interface-guidelines/ios/visual-design/typography/#dynamic-type-sizes default style -->
+	<FontFamily x:Key="CupertinoFontFamily">SF Pro</FontFamily>
+
+	<!--  Based on https://developer.apple.com/design/human-interface-guidelines/ios/visual-design/typography/#dynamic-type-sizes default style  -->
 	<Style x:Key="CupertinoBaseTextBlockStyle"
 		   TargetType="TextBlock">
 		<Setter Property="TextTrimming"
 				Value="CharacterEllipsis" />
 		<Setter Property="TextWrapping"
 				Value="Wrap" />
 		<Setter Property="FontFamily"
-				Value="{StaticResource SFProFontFamily}" />
+				Value="{StaticResource CupertinoFontFamily}" />
 	</Style>
 
 	<!--
-	******************************************************************
+		******************************************************************
 		STYLE GUIDE TEXTBLOCKS STYLES
-	******************************************************************
+		******************************************************************
 	-->
 
-	<!-- Large Title -->
-	<Style x:Key="LargeTitleTextBlockStyle"
+	<!--  Large Title  -->
+	<Style x:Key="CupertinoLargeTitle"
 		   TargetType="TextBlock"
 		   BasedOn="{StaticResource CupertinoBaseTextBlockStyle}">
 
 		<Setter Property="FontSize"
-				Value="45" />
+				Value="34" />
 		<Setter Property="LineHeight"
-				Value="55" />
+				Value="41" />
```

Comment:
Those styles are not as designed on Figma. Are you sure about these changes?

Reply:
These are the numbers coming from the Human Interface Design Guidelines, I trust those more than the Figma file.